tests/modules/diamond/blastx/main.nf

```diff
@@ -9,18 +9,20 @@ workflow test_diamond_blastx {
 
     db = [ file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true) ]
     fasta = [ file(params.test_data['sarscov2']['genome']['transcriptome_fasta'], checkIfExists: true) ]
-    outext = 'txt'
+    out_ext = 'tfdfdt'
```

Comment:
```suggestion
    out_ext = 'tfdfdt'  // Nonsense file extension to check default case.
```

Reply:
Ah yes, forgot to revert that :laughing: , keep it in makes sense anywhay :+1: